permissions/Base.permission

```diff
@@ -18,6 +18,9 @@ writable-run-user
 # we use "private-etc selinux" -> must not blacklist it
 noblacklist /etc/selinux
 
+# we need to be able to whitelist it later, while it's in disable-common.inc
+noblacklist ${HOME}/.gnupg
```

Comment:
Need to double-check, but ...

When "blacklist", "noblacklist", "whitelist" etc directives are read from config files, they are just queued for later processing - in fixed per directive order -> it should be so that the .gnupg directory can remain blacklisted by default, and both "noblacklist" and "whitelist" can be in feature/app specific config file -> depending on what the intent is, "noblacklist" can/should be moved to GnuPG.permission in this case.

Reply:
It seems to me that a blacklist order when arriving before a noblacklist is taking precedence. I mean, we cannot revert a blacklisted entry after it is done. I cannot find the documentation for that again... The `/etc/firejail/gpg.profile` is doing it the same way : first you noblacklist to cancel later blacklisting in including `disable-common.inc`.

I'm quite sure, I tested to put the `noblacklist` in the GnuPG.permission file, without touching the Base one, but without any success.